noxfile.py

```diff
@@ -304,7 +304,16 @@ def showcase_library(
             f"{tmp_dir}/testing/constraints-{session.python}.txt"
             )
             # Install the library with a constraints file.
-            session.install("-e", tmp_dir, "-r", constraints_path)
+            if session.python == "3.7":
+                session.install("-e", tmp_dir, "-r", constraints_path)
+                if rest_async_io_enabled:
+                    # NOTE: We re-install `google-api-core` and `google-auth` to override the respective
+                    # versions for each specified in constraints-3.7.txt. This is needed because async REST
+                    # is not supported with the minimum version of `google-api-core` and `google-auth`.
+                    session.install('--no-cache-dir', '--force-reinstall', "google-api-core")
```

Comment:
Since the 3.7 constraints file is meant to address the minimum version of our dependencies, could we have these installs hard code the minimum version of `google-api-core` and `google-auth` that work with async REST?

Reply:
addressed.